ceremonies/2020/r4-key.yaml

```diff
@@ -8,4 +8,5 @@ key:
   type: rsa
   rsa-mod-length: 2048
 outputs:
-  public-key-path: /run/shm/ceremonies/2020/int-r4.key.pem
\ No newline at end of file
+  public-key-path: ./int-r4.key.pem
+  
```

Comment:
I... what... github thinks there's an empty line at the bottom of the file, and *also* thinks that there's no trailing newline at the end of that line. I have never seen this before.
![image](https://github.com/letsencrypt/ceremony-demos/assets/2474926/9608a62d-eabd-4e9c-bab1-c388447ecbee)